x/incentives/types/msgs.go

```diff
@@ -36,10 +36,13 @@ func (m MsgCreateGauge) Type() string { return TypeMsgCreateGauge }
 
 // ValidateBasic checks that the create gauge message is valid.
 func (m MsgCreateGauge) ValidateBasic() error {
+	lockType := m.DistributeTo.LockQueryType
+
 	if m.Owner == "" {
 		return errors.New("owner should be set")
 	}
-	if sdk.ValidateDenom(m.DistributeTo.Denom) != nil {
+	// For no lock type, the denom must be empty and we check that down below.
+	if lockType != lockuptypes.NoLock && sdk.ValidateDenom(m.DistributeTo.Denom) != nil {
 		return errors.New("denom should be valid for the condition")
 	}
```

Comment:
Can we fix this error message to be more descriptive? I know its left over from previous

Reply:
done